generator/src/test/java/com/scottlogic/deg/generator/cucumber/After.feature

```diff
@@ -41,19 +41,17 @@ Scenario: 'After' non-existent leap year date fails with error
 
   ### after ###
 
-# Defect 611 "before run against a non contradicting before results in an error" related to this scenario
-@ignore
 Scenario: 'After' with a non-contradicting 'After' is successful
   Given foo is after 2019-01-01T00:00:00.000
     And foo is after 2019-02-01T00:00:00.000
     And the generator can generate at most 5 rows
   Then the following data should be generated:
     | foo                     |
     | 2019-02-01T00:00:00.001 |
-    | 2019-02-01T00:00:00.002 |
-    | 2019-02-01T00:00:00.003 |
-    | 2019-02-01T00:00:00.004 |
-    | 2019-02-01T00:00:00.005 |
+    | 2020-02-01T00:00:00.001 |
+    | 2021-02-01T00:00:00.001 |
+    | 2022-02-01T00:00:00.001 |
+    | 2023-02-01T00:00:00.001 |
```

Comment:
start at 2018-01-01T00:00:00.000 and work down dates at a granularity of milisecond.